README.md

```diff
@@ -827,6 +827,7 @@ These colorschemes may not specialize in Tree-sitter directly but are written in
 - [MarcHamamji/runner.nvim](https://github.com/MarcHamamji/runner.nvim) - A customizable Lua code runner.
 - [google/executor.nvim](https://github.com/google/executor.nvim) - Allows you to run command line tasks in the background and be notified of results.
 - [Zeioth/compiler.nvim](https://github.com/Zeioth/compiler.nvim) - Compiler for building and running your code without having to configure anything.
+- [Zeioth/makeit.nvim](https://github.com/Zeioth/makeit.nvim) - Neovim Makefile plugin.
```

Comment:
```suggestion
- [Zeioth/makeit.nvim](https://github.com/Zeioth/makeit.nvim) - Makefile utilities and UI.
```

Reply:
`Makefile runner based on overseer.`

I've changed it to this in order to
* Avoid using the words `plugin` or `neovim`.
* Be concise about what the plugin does.

https://github.com/Zeioth/awesome-neovim/blob/3765330507ea4704c685a8d6b0ff1f92e665b7ae/README.md?plain=1#L830